content/200-concepts/100-components/01-prisma-schema/index.mdx

```diff
@@ -19,7 +19,7 @@ Whenever a `prisma` command is invoked, the CLI typically reads some information
 - `prisma generate`: Reads _all_ above mentioned information from the Prisma schema to generate the correct data source client code (e.g. Prisma Client).
 - `prisma migrate dev`: Reads the data sources and data model definition to create a new migration.
 
-You can also [use environment variables](#using-environment-variables) inside the schema file to provide configuration options when a CLI command is invoked.
+You can also [use environment variables](#accessing-environment-variables-from-the-schema) inside the schema file to provide configuration options when a CLI command is invoked.
```

Comment:
I think we use the `#using-environment-variables` in many places, I guess it doesn't work anymore?

Reply:
Its to a specific place. I moved that doc. Hmm givent he recent issues with links moving about I will do a sweep of the docs to make sure that still works

Reply:
Please ping @mhwelander about disappearing anchors. She will know if we already have a solution for that or not (and take a not that this is again popping up).